tutorial/README.md

```diff
@@ -134,12 +132,11 @@ $ dcos package install beta-tensorflow --options=./helloworld.json
 ```
 Congratulations, you have just trained your first custom TensorFlow Model!
 
-**NOTE:** Once the example has completed, the `ps-*` and `*worker-*` tasks will terminate, but the `EXAMPLE_NAME` scheduler may still remain. This can be removed (or the computation terminated) by running:
+**NOTE:** Once the example has completed, the `parameter-server-*` and `*worker-*` tasks will terminate, but the scheduler may still remain. This can be removed (or the computation terminated) by running:
 ```bash
-$ dcos package uninstall beta-tensorflow --app-id=/EXAMPLE_NAME
+$ dcos package uninstall beta-tensorflow --app-id=helloworld
```

Comment:
I believe this should actually be `--app-id=/helloworld`. That's my bad for typing it without the slash in my previous comment.